cobbler/cobbler_collections/collection.py

```diff
@@ -506,8 +517,7 @@ def add(
                         ref.enable_menu = False
                     self.lite_sync.add_single_profile(ref)
                     self.api.sync_systems(
-                        systems=self.find(
-                            "system",
```

Comment:
This is a bug. I included a fix for it in this PR, but we need to remember to backport it to 3.3.5.